documentation/docs/packages/analytics.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 1
+---
+
+# analytics
+Monk analytics abstraction package.
+
+![npm latest package](https://img.shields.io/npm/v/@monkvision/analytics/latest.svg)
+
+## Overview
+This package provides an abstraction layer for the analytics features in the MonkJs ecosystem. If you plan on using any
+of these features, you can use this package to properly set up the analytics inside your application.
+
+In the MonkJs SDK, we use this "abstraction" package as a way of offering the possibility to developers to choose their
+own Analytics tools and platform if they don't want to use Posthog (the platform used internally by Monk).
+
+## Complete Documentation
+As every other package in the SDK, please refer to
+[its README file](https://github.com/monkvision/monkjs/blob/main/packages/analytics/README.md) for a complete
+documentation on how this package works.
```

Comment:
Once I will create the PR for the documentation and will have merged it, you'll need to update this page so that :
- It has the proper sidebar position (right after the monitoring page)
- It is more complete and detailed 
- It looks more like the monitoring page